gilbreth_support/urdf/simple_gripper/simple_gripper.urdf.xacro

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
```

Comment:
This is a new gripper. Because the gripper which we are using now has a very small tip. Our current output is not very precise and robust, most of the time the small tip will miss the target. The new gripper is bigger and has a more robust behavior than the origin one.

Reply:
Unless you think we would wanna switch between grippers I don't see a reason to have two or more of them.  I'm OK if you keep this simple one and delete the old one or you can also just modify the old one as well.